src/components/customerPortal/PortalInvoicesList.tsx

```diff
@@ -139,48 +230,91 @@ const PortalInvoicesList = ({ translate, documentLocale }: PortalCustomerInvoice
     },
   })
 
-  const { debouncedSearch, isLoading } = useDebouncedSearch(getInvoices, loading)
+  const { debouncedSearch, isLoading: searchIsLoading } = useDebouncedSearch(getInvoices, loading)
   const { metadata, collection } = data?.customerPortalInvoices || {}
   const hasSearchTerm = !!variables?.searchTerm
   const hasNoInvoices = !loading && !error && !metadata?.totalCount && !hasSearchTerm
 
+  const { currentPage = 0, totalPages = 0 } = metadata || {}
+
+  const refreshSection = () => {
```

Comment:
Removed.

To give some additional context on why this was introduced in the first place: if we pass the `refetch` from Apollo directly to an onClick handler, it results in a 'cannot convert circular structure' error (https://stackoverflow.com/questions/41668454/circular-error-on-apollo-client-when-trying-to-refetch). Not sure why, but it would be interesting to debug.

Reply:
hum interesting, and might be in their side or I don't get it.
Yeah ok both would have worked then, i guess